src/motion/integrator.F

```diff
@@ -1148,6 +1151,338 @@ SUBROUTINE nvt(md_env, globenv)
 
    END SUBROUTINE nvt
 
+! **************************************************************************************************
+!> \brief npt_ia integrator for particle positions & momenta
+!>      isotropic box changes
+!> \param md_env ...
+!> \param globenv ...
+!> \par History
+!>      none
+!> \author CJM
+! **************************************************************************************************
+   SUBROUTINE npt_ia(md_env, globenv)
```

Comment:
Maybe instead of duplicating the code you could just add flag to the `npt_i` routine below?